build.gradle

```diff
@@ -948,6 +948,48 @@ project(':iceberg-snowflake') {
   }
 }
 
+project(':iceberg-tencentcloud') {
+  test {
+    useJUnitPlatform()
+  }
+
+  dependencies {
+    implementation project(path: ':iceberg-bundled-guava', configuration: 'shadow')
+    api project(':iceberg-api')
+    implementation project(':iceberg-core')
+    implementation project(':iceberg-common')
+
+    compileOnly (libs.cos.sdk) {
+      exclude group: 'com.google.code.gson', module: 'gson'
+    }
+    compileOnly (libs.cos.sdk.bundle) {
+      exclude group: 'com.google.code.gson', module: 'gson'
+    }
+    compileOnly libs.jaxb.api
+    compileOnly libs.activation
+    compileOnly libs.jaxb.runtime
+    compileOnly(libs.hadoop2.common) {
+      exclude group: 'org.apache.avro', module: 'avro'
+      exclude group: 'org.slf4j', module: 'slf4j-log4j12'
+      exclude group: 'javax.servlet', module: 'servlet-api'
+      exclude group: 'com.google.code.gson', module: 'gson'
+    }
+
+    testImplementation libs.jackson.dataformat.xml
+    testImplementation project(path: ':iceberg-api', configuration: 'testArtifacts')
+    testImplementation libs.spring.web
```

Comment:
why are all these spring dependencies defined here? I haven't seen any usage of those in the tests. I'd suggest to remove dependencies that aren't necessary and slim this down to the absolute minimum